build.js

```diff
@@ -21,6 +22,10 @@ async function main() {
   await server.start(9001);
   await page.goto('http://localhost:9001/');
   await page.waitForSelector('.passing');
+
+  const chartContent = await generateChartSVG();
+  await page.evaluate((content) => document.querySelector('#chart').innerHTML = content, chartContent);
```

Comment:
For the record, what I had in mind was a little different. I was thinking we'd simply add the chart JS to `main.mjs` where we already have access to the data. The build script would then not need any changes.